docs-source/content/faq/oci-lb.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Using an OCI Load Balancer"
+date: 2019-09-25T12:41:38-04:00
+draft: false
+---
+
+If you are running your Kubernetes cluster on Oracle Container Engine 
+for Kubernetes (commonly known as OKE), you can have OCI automatically
+provision load balancers for you by creating a `Service` of type
+`LoadBalancer` instead of (or in addition to) installing an
+ingress controller like Traefik or Voyager.
+
+OKE Kubernetes worker nodes normally do not have public IP addresses.
+This means that the `NodePort` services created by the operator are 
+not usable, since they would expose ports on the worker node's public IP
+address.  Instead, you can use an OCI Load Balancer to provide access
```

Comment:
OCI Load Balancer -> OCI load balancer (unless it's a product name, then capitalize it consistently)